### ISSUE TITLE
it's broken

### ISSUE THREAD

**Issue by mlivneh** [arturkot/the-house-game#8]
downloaded it, stuck in loading.
the other pages are dead
latest chrome

**Comment by aiwongs** [comment]
me too, the source is not working in my env.

**Comment by arturkot** [comment]
I'll try looking into it over the weekend.

**Comment by arturkot** [comment]
Hello @mlivneh and @aiwongs ,

TL;DR; you _probably_ didn't run the game using [some kind](https://www.mamp.info) of [local server](https://github.com/http-party/http-server).

---

Here are steps I take to run the game locally:

## Computer

1. I clone/copy the repository.
2. **Run local server**.
  - It's a crucial step, the game won't run when opened directly from the file system.
  - The server doesn't need to run from inside the game's directory. As long as the game's directory can be reached with an URL, the game is going to work fine. E.g `http://localhost:8080/some/path/the-house-game/` will work.
  - Any server able to serve static files over http should work.
3. Enter the address in your browser. It's going to be different depending on what tool you're using. E.g., `http://localhost:3000`, `http://localhost:4200/the-house-game`, etc.

## iPad

1. Cloned the repo with [Working Copy](https://workingcopyapp.com/).
2. Opened `index.html` file in the game's directory with [Textastic](https://www.textasticapp.com/).
3. [Tapped  the "glasses" icon to preview `index.html`](https://www.textasticapp.com/v9/manual/viewing_editing_files/web_preview.html?highlight=preview) in the built-in Textastic server.

**Comment by aiwongs** [comment]
> Hello @mlivneh and @aiwongs ,
> 
> TL;DR; you _probably_ didn't run the game using [some kind](https://www.mamp.info) of [local server](https://github.com/http-party/http-server).
> 
> Here are steps I take to run the game locally:
> 
> ## Computer
> 1. I clone/copy the repository.
> 2. **Run local server**.
> 
> * It's a crucial step, the game won't run when opened directly from the file system.
> * The server doesn't need to run from inside the game's directory. As long as the game's directory can be reached with an URL, the game is going to work fine. E.g `http://localhost:8080/some/path/the-house-game/` will work.
> * Any server able to serve static files over http should work.
> 
> 3. Enter the address in your browser. It's going to be different depending on what tool you're using. E.g., `http://localhost:3000`, `http://localhost:4200/the-house-game`, etc.
> 
> ## iPad
> 1. Cloned the repo with [Working Copy](https://workingcopyapp.com/).
> 2. Opened `index.html` file in the game's directory with [Textastic](https://www.textasticapp.com/).
> 3. [Tapped  the "glasses" icon to preview `index.html`](https://www.textasticapp.com/v9/manual/viewing_editing_files/web_preview.html?highlight=preview) in the built-in Textastic server.

Thank u for your reply, it works fine, I now FORK your project, I have created the Chinese branch these days, this project is so great. ：）
